tests/system/migrations/test_0055_remove_statutes.py

```diff
@@ -0,0 +1,1394 @@
+            "content_object_id": "motion/1",
+            "parent_id": 1,
+            "child_ids": [3],
+            "tag_ids": [2],
+            "projection_ids": [4],
+            "meeting_id": 11,
+        },
+        "agenda_item/3": {
+            "id": 3,
+            "content_object_id": "topic/1",
+            "parent_id": 2,
+            "child_ids": [],
+            "tag_ids": [1],
+            "projection_ids": [5],
+            "meeting_id": 11,
+        },
+        "topic/1": {
+            "id": 1,
+            "title": "A tropical topic",
+            "agenda_item_id": 3,
+            "projection_ids": [6],
+            "meeting_id": 11,
+        },
+        "list_of_speakers/1": {
+            "id": 1,
+            "content_object_id": "motion/1",
+            "speaker_ids": [1],
+            "structure_level_list_of_speakers_ids": [1],
+            "projection_ids": [7],
+            "meeting_id": 11,
+        },
+        "structure_level_list_of_speakers/1": {
+            "id": 1,
+            "structure_level_id": 1,
+            "list_of_speakers_id": 1,
+            "initial_time": 30,
+            "speaker_ids": [1],
+            "meeting_id": 11,
+        },
+        "structure_level/1": {
+            "id": 1,
+            "name": "ErstePartei",
+            "color": "#FF0000",
+            "default_time": 30,
+            "structure_level_list_of_speakers_ids": [1],
+            "meeting_id": 11,
+        },
+        "speaker/1": {
+            "id": 1,
+            "meeting_user_id": 1,
+            "point_of_order_category_id": 1,
+            "list_of_speakers_id": 1,
+            "meeting_id": 11,
+        },
+        "point_of_order_category/1": {
+            "id": 1,
+            "text": "A point of order category",
+            "speaker_ids": [1],
+            "rank": 1,
+            "meeting_id": 11,
+        },
+        "motion_workflow/1": {
+            "id": 1,
+            "default_statute_amendment_workflow_meeting_id": 11,
+            "state_ids": [1],
+            "meeting_id": 11,
+        },
+        "motion_statute_paragraph/1": {
+            "id": 1,
+            "title": "string",
+            "text": "HTML",
+            "meeting_id": 11,
+            "motion_ids": [1],
+        },
+        "motion_statute_paragraph/2": {
+            "id": 2,
+            "title": "string",
+            "text": "HTML",
+            "meeting_id": 11,
+            "motion_ids": [2],
+        },
+        "motion_state/1": {
+            "id": 1,
+            "name": "string",
+            "workflow_id": 1,
+            "motion_recommendation_ids": [1, 2],
+            "meeting_id": 11,
+        },
+    }
+    write(
+        *[
+            {
+                "type": "create",
+                "fqid": fqid,
+                "fields": fields,
+            }
+            for fqid, fields in data.items()
+        ]
+    )
+    return data
+
+
+def test_no_delete_without_statute(write, finalize, assert_model):
```

Comment:
This test needs to contain the other affected models as well to see if nothing happens to them either.

Reply:
Would probably be easier to just use the comprehensive data, with normal motions instead of statutes, run the migration and see if the data in the backend is still the same afterwards.

This could be done generically if the comprehensive data function returned the dict it creates.
Then you could just circle through the entries, generating one assert call per fqid.

Reply:
I think that 'test_no_sideffects_submodels' should already do this. So I will just delete this test.